SCClassLibrary/Common/GUI/Base/QWebView.sc

```diff
@@ -253,6 +253,27 @@ WebView : View {
 	contentsSize { ^this.getProperty('contentsSize') }
 
 	scrollPosition { ^this.getProperty('scrollPosition') }
+    scrollPosition_ { |point| ^this.setProperty('scrollPosition', (point / this.zoom)) }
+
+    scrollUp { |value|
+        var position = this.scrollPosition;
+        this.scrollPosition_(
+            Point(
+                position.x,
+                (position.y -  value ).max(0)
```

Comment:
Please remove double-space and trailing space: `(position.y - value).max(0)`

Reply:
should be ok now.